src/main/java/net/greghaines/jesque/worker/WorkerImpl.java

```diff
@@ -529,6 +529,8 @@ protected void process(final Job job, final String curQueue) {
             final Object instance = this.jobFactory.materializeJob(job);
             final Object result = execute(job, curQueue, instance);
             success(job, instance, result, curQueue);
+        } catch (DontPerformException e) {
+            // Just abort the job. Job is not considered a failure nor a success.
```

Comment:
I read through the Ruby implementation and I see how they're just dropping the job on the floor but that doesn't sit well with me. At bare minimum I would want to see a count of aborted jobs as well as a new event, i.e.:
```java
this.jedis.incr(key(STAT, ABORTED));
this.jedis.incr(key(STAT, ABORTED, this.name));
this.listenerDelegate.fireEvent(JOB_ABORTION, this, curQueue, job, null, null, e);
```